blink/README.md

```diff
@@ -0,0 +1,40 @@
+# BlinkOps Integration
+
+## Overview
+
+The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual drag-and-drop editor to quickly automate your Datadog workspace, or use one of the hundreds of existing pre-made automations from the [library][2].
```

Comment:
Can you clarify on what you mean by drag and drop editor? We want to make sure the description is as clear as possible for customers.

Reply:
We allow users to drag and drop "actions" into a automation which represents a workflow, so our integration with Datadog is that we have Datadog actions that you can insert. I added some clarification to the readme about this but let me know if it is still unclear